src/components/Reviews.jsx

```diff
@@ -30,17 +43,57 @@ export default function Reviews() {
     );
   }
 
+  const handleChange = (event) => {
+    setSortType(event.target.value);
+  };
+
+  const handleRadioChange = (event) => {
+    setSortOrder(event.target.value);
+  };
+
   return (
     <Box id="all-page" sx={{ width: "100%" }}>
       <header id="all-reviews">
-        <h2 >All Reviews</h2>
+        <h2>All Reviews</h2>
         <Tooltip id="filter-tip" title="Click here to filter by category.">
           <Link to="/api/categories">
             <IconButton>
               <FilterListIcon />
             </IconButton>
           </Link>
         </Tooltip>
+
+        <Box sx={{ minWidth: 120 }}>
+          <FormControl fullWidth>
+            <InputLabel id="demo-simple-select-label">Sort By</InputLabel>
+            <Select
+              labelId="demo-simple-select-label"
+              id="demo-simple-select"
+              value={sortType}
+              label="Sort By"
+              onChange={handleChange}
+            >
+              <MenuItem value={"votes"}>Votes</MenuItem>
+              <MenuItem value={"created_at"}>Date</MenuItem>
+              <MenuItem value={"comment_count"}>Comments</MenuItem>
+            </Select>
+          </FormControl>
+        </Box>
+
+        <FormControl>
+          <RadioGroup row defaultValue="desc">
```

Comment:
This sorting feature is great, but we also want to make sure that it works for both all reviews and reviews by category.